src/status_im2/contexts/shell/jump_to/components/home_stack/view.cljs

```diff
@@ -3,12 +3,12 @@
             [status-im.ui.screens.wallet.accounts.views :as wallet.accounts]
             [status-im2.contexts.wallet.home.view :as wallet-new]
             [status-im2.contexts.chat.home.view :as chat]
+            [status-im2.contexts.browser.home.view :as browser-new]
```

Comment:
I named it ´browser-new´ just to be consistent with the example you sent, but since we will just replace the old browser I believe we could just name it ´browser´. WDYT?

Reply:
yep, just browser sounds good here 👌